stories/typography/dice.js

```diff
@@ -83,7 +85,7 @@ import { Dice } from '@aller/shiny';
 						</tr>
```

Comment:
Whats the deal with this syntax from github? Check up that it looks ok in editor!

Reply:
Looks ok in editor, just a github wtf